content/docs/tasks/security/vault-ca/index.md

```diff
@@ -154,3 +158,123 @@ between workloads using the certificates the Vault CA issued.
 After completing this tutorial, you may delete the testing cluster created
 at the beginning of this tutorial.
 
+## Configuring a basic Vault server
+
+The Vault server administrator is responsible of hosting and managing the server. When signing a CSR, Istio Citadel Agent
+passes the Kubernetes service account token of the workload and the CSR to the Vault server,
+which authenticates and authorizes the request (e.g., based on the
+[Vault Kubernetes `auth` method](https://www.vaultproject.io/docs/auth/kubernetes.html)) and
+returns the signed certificate to Istio Citadel Agent, if the request is authorized.
+Based on their security requirements, owners of Vault servers may configure various
+certificate issuance policies for Kubernetes service accounts and certificate
+signing requests.
+
+The following instructions configure an example basic Vault server to authenticate and authorize a CSR
+based on the Vault Kubernetes auth method.
+**The instructions here are for illustrative purposes only. Please consult with security experts
+to set up the security configuration and certificate issuance policies of your Vault servers.** The instructions are based on the
+posts [1](https://evalle.xyz/posts/integration-kubernetes-with-vault-auth/) and
+[2](https://coreos.com/tectonic/docs/latest/vault-operator/user/kubernetes-auth-backend.html).
```

Comment:
This documentation is moving to redheat.

Reply:
Thanks! The links have been updated based on the deprecation notice on the original webpages.